grabbit/src/integrationTest/groovy/com/twcable/grabbit/server/services/ServerServiceSpec.groovy

```diff
@@ -72,7 +72,7 @@ class ServerServiceSpec extends Specification {
 
         when:
         //This will also actually execute the Batch Job internally
-        syncServerService.getContentForRootPath("/default.groovy", (Collection<String>)Collections.EMPTY_LIST, "", mockServletOutputStream)
+        syncServerService.getContentForRootPath("", "/default.groovy", (Collection<String>)Collections.EMPTY_LIST, "", mockServletOutputStream)
```

Comment:
Yeah I should pass in `"admin"` instead of `""` :) For an `empty` username I believe Jackalope internally defaults to anonymous user that's why the test passes.